resources/js/entities/User.js

```diff
@@ -31,11 +31,11 @@ export default class User {
 
   set lastName(value) {
     const type = typeof value;
-    if (type === "string") {
-      this.__lastName = value;
-    } else {
+    if (type !== "string") {
       throw new TypeError(`String expected, got ${type}.`);
     }
+
+    this.__lastName = value;
```

Comment:
Un detalle de tipeo. En el Constructor en atributo está escrito como 'this._lastName' y en esta linea como 'this.__lastName'. Si no me equivoco en el getter de esta propiedad  sucede lo mismo.

Reply:
gracias ahi lo arreglo